applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -324,6 +330,17 @@ perform_participant_action(Conference, ?KICK, ParticipantId) ->
     kapps_conference_command:kick(ParticipantId, Conference).
 
 %% add real-time call-info to participants
+-spec enrich_participant(ne_binary(), ne_binary(), cb_context:context()) -> cb_context:context().
+enrich_participant(ParticipantId, ConferenceId, Context) ->
+    Participants = extract_participants(
+                     request_conference_details(ConferenceId)
+                    ),
+    [Normalized|_] = [kz_json:normalize_jobj(JObj)
+                      || JObj <- request_call_details(Participants)
+                             ,kz_json:get_binary_value(<<"Participant-ID">>, JObj) == ParticipantId
```

Comment:
So everywhere else ParticipantId is considered an integer(). I'd prefer get_integer_value/2 & integer() in the specs, but that's up for debate…